tokio/Cargo.toml

```diff
@@ -60,6 +60,7 @@ net = [
   "winapi/minwindef",
 ]
 process = [
+  "fs", # TODO: remove this and expose the internals privately
```

Comment:
This needs to be removed and the `fs` module exposed only internally like we do for other features. I wanted to post a WIP branch before finishing everything